src/index.ts

```diff
@@ -59,8 +59,7 @@ const COMMANDS: { [key: string]: [(...args) => void, string, string] } = {
   'get-ephemeral-used': [c.getEphemeralUsed, '', 'show used ephemeral addresses'],
   'get-ephemeral-address-privkey': [c.getEphemeralPrivKey, '<index>', 'get private key for concrete ephemeral address'],
   'wipe-state': [c.cleanState, '', 'wipe internal state and history'],
-  'enable-delegated-prover': [c.enableDelegatedProver, '', 'enable delegated prover usage'],
-  'disable-delegated-prover': [c.disableDelegatedProver, '', 'disable delegated prover usage'],
+  'set-prover-mode': [c.setProverMode, '<mode>', 'set prover mode'],
```

Comment:
Please provide available modes here (the user have no chance to guess string values)

Reply:
Done in https://github.com/zkBob/zkbob-console/pull/38/commits/4b2707d15421db160eca6db43a75e6151132f0fb